geozero/src/wkt/wkt_reader.rs

```diff
@@ -7,34 +7,34 @@ use wkt::Geometry;
 
 /// WKT String.
 #[derive(Debug)]
-pub struct WktString(pub String);
+pub struct WktString<B: AsRef<[u8]>>(pub B);
```

Comment:
I agree that it's nice to be able to take either a String or a &str, but I think it's confusing to explain why we have two different functionally equivalent types like this.

What would you think about unifying them? Either by deprecating one or (my preference) revert and deprecate both legacy types and introduce a third unified struct (maybe called `Wkt`?).

(same for Ewkt)

Reply:
I agree that having multiple redundant types is confusing; I wasn't sure which would be preferred, but I'm happy to deprecate these and move to `Wkt`

> I agree that it's nice to be able to take either a String or a &str

It's also nice that it can take in a `Vec<u8>` and `&[u8]` at the same time! There were a couple places that previously had copies with `to_vec` that could be removed